schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
-);
+  id INTEGER,
+  name VARCHAR(50),
+  date_of_birth DATE,
+  escape_attempts INTEGER,
+  neutered BOOLEAN,
+  weight_kg DECIMAL,
+  species VARCHAR(50)
```

Comment:
**Changes Required**
- [ ] Since animal table already existed before, we should not update it directly by add coulmns. We should rather use the `ALTER` SQL command to add the missing columns.